src/LinearSolvers/LinearSolvers.jl

```diff
@@ -164,14 +156,26 @@ function ldiv!(solver::DQGMRES,
     y::AbstractVector, J::AbstractMatrix, x::AbstractVector,
 )
     P = solver.precond.P
-    @timeit solver.timer "GPU-DQGMRES" begin
-        (y[:], status) = Krylov.dqgmres(J, x, M=P, memory=solver.memory)
-    end
+    (y[:], status) = Krylov.dqgmres(J, x, N=P, memory=solver.memory)
+    return length(status.residuals)
+end
+
+struct KrylovBICGSTAB <: AbstractIterativeLinearSolver
```

Comment:
Nice we now have `KrylovBICGSTAB` in the package

Reply:
The fastest of them all!